eslint/babel-eslint-parser/src/configuration.cjs

```diff
@@ -11,7 +11,7 @@ exports.normalizeESLintConfig = function (options) {
 
   return {
     babelOptions: { cwd: process.cwd(), ...babelOptions },
-    ecmaVersion,
+    ecmaVersion: ecmaVersion === "latest" ? 2020 : ecmaVersion,
```

Comment:
The latest is 2022 as of August 2021. I think we can normalize `"latest"` to `1e8` like acorn did (https://github.com/acornjs/acorn/blob/691a31b1e3d6936f4e6991da3d81c1a6b06610cf/acorn/src/options.js#L109) so we don't have to update it every year.

Reply:
Good idea.

Reply:
I won't miss the date.